base/common/src/main/java/com/netscape/certsrv/system/AdminSetupResponse.java

```diff
@@ -31,6 +33,7 @@
 public class AdminSetupResponse {
 
     @XmlElement
+    @JsonValue
```

Comment:
So the CI failed because it couldn't find an `adminCert` in the response object returned by `setupAdmin()`:
https://github.com/dogtagpki/pki/blob/master/base/server/python/pki/server/deployment/__init__.py#L947

The `setupAdmin()` got the response from `/rest/installer/setupAdmin` REST API:
https://github.com/dogtagpki/pki/blob/master/base/common/python/pki/system.py#L421-L424

That API was returning an `AdminSetupResponse` object:
https://github.com/dogtagpki/pki/blob/master/base/server/src/main/java/org/dogtagpki/server/rest/SystemConfigService.java#L202-L206

So the JSON mapping for `AdminSetupResponse` needs to match the structure expected by the client:
```
{
    adminCert: { ... }
}
```
I think we should use `@JsonValue` only if the object contains just a single value and the value is simple enough (e.g. cert ID) instead of a complex object. In this case we could either (a) use the regular JSON mapping, or (b) change the client to recognize the new structure, but considering it's a `Response` object it might have more fields in the future so I'd suggest option (a) for now.

Reply:
Interesting, I did try a) but maybe I did it wrong :thinking: I'll give it another go.

Reply:
Judging by the tests passing, my original attempt to resolve the error by doing 1) appears to pass CI - which leads me to believe that it was my local PKI setup that was broken...